java/configuration/src/main/java/sleeper/configuration/properties/UserDefinedInstanceProperty.java

```diff
@@ -977,6 +983,62 @@ public interface UserDefinedInstanceProperty extends InstanceProperty {
             .validationPredicate(Utils::isPositiveInteger)
             .propertyGroup(InstancePropertyGroup.DEFAULT).build();
 
+    UserDefinedInstanceProperty DEFAULT_INGEST_BATCHER_MIN_JOB_SIZE = Index.propertyBuilder("sleeper.default.ingest.batcher.job.min.size")
+            .description("Specifies the minimum file size required for an ingest job to be batched and sent. " +
```

Comment:
Maybe "minimum total file size"? Saying "minimum file size" makes it sound like it's one file rather than the batch.

Reply:
Done, thanks